entity-types/infra-awscloudwatchapplicationsignals/golden_metrics.yml

```
@@ -0,0 +1,30 @@
+requests:
+  title: Top operations by requests
+  unit: COUNT
+  queries:
+    aws:
+      select: count(aws.applicationsignals.Fault)
```

```suggestion
      select: count(`aws.applicationsignals.Fault`)
```

not sure, just comparing with the other 2